### ISSUE TITLE
help: Lua Plugins Not Working With `help PluginName` Syntax

### ISSUE THREAD

**Issue by D1G1T4L3CH0** [DeathCradle/Terraria-s-Dedicated-Server-Mod#103]
Expected result:
```
help Renworld
Renames a world.
Usage:    renworld <\"World Name\"> [-nofile] [-nosave]
```

Actual result:
```
help RenWorld
help: No such command: RenWorld
```

This is in my plugin:
```
    AddCommand("renworld")
        :WithAccessLevel(AccessLevel.CONSOLE)
        :WithDescription("Renames a world.")
        :WithHelpText("Usage:    renworld <\"World Name\"> [-nofile] [-nosave]")
        :WithPermissionNode("RenWorld.command")
        :LuaCall(export.RenameWorld)
```

**Comment by DeathCradle** [comment]
Fixed in de044b48ca55d003d3bda602b2f1ca67fe01f678

**Comment by D1G1T4L3CH0** [comment]
resolved
